distribution/openrct2.d.ts

```diff
@@ -3324,6 +3324,20 @@ declare global {
          * The total number of frames in the current animation.
          */
         readonly animationLength: number;
+
+        lawnsMown: number | null;
+
+        gardensWatered: number | null;
+
+        litterSwept: number | null;
+
+        binsEmptied: number | null;
+
+        ridesFixed: number | null;
+
+        ridesInspected: number | null;
+
+        vandalsStopped: number | null;
```

Comment:
While I think these speak for themselves, please add a brief comment for each of them. It would be good to indicate to which staff members they apply, for example.

Reply:
This is connected to our discussion above, whether we should split the interface up into one per staff type.